src/components/Chart.jsx

```diff
@@ -70,10 +71,11 @@ class Chart extends Component {
         const insideSubHolder = array.filter(c => /(TradeStart)|(TradeEnd)/.test(c.type.displayName));
         const renderTopWidgets = topWidgets || defaultTopWidgets;
 
-
         const defaultTheme = (setting && setting.theme) ? setting.theme : 'light';
         const defaultCandleCountdown = (setting && setting.countdown) ? setting.countdown : false;
 
+        document.getElementById('root').style.setProperty('--view-height', window.innerHeight +'px');
```

Comment:
@MahboobehMohammadi I may be wrong but i think it only works on `/app`, we don't have a `#root` element in binary-static.

Reply:
And the second thing is the use of css variables, which is a fantastic idea, but still pretty new and only supported by latest browser versions.